### PR TITLE
feat(electron): Removed old check updates system and use the new one

### PR DESCRIPTION
**Pull request by dragonDScript**
- [x] Tested the feature and it's working on a current and clean install.
- [x] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

File: src/state/GlobalState.tsx [hunk 1]
```diff
@@ -266,21 +266,9 @@ export class GlobalState extends PureComponent<Props> {
   }
 
   checkVersion = async () => {
-    const { t } = this.props
     const newVersion = await ipcRenderer.invoke('checkVersion')
     if (newVersion) {
-      const { response } = await ipcRenderer.invoke('openMessageBox', {
```
Comment (flavioislima) [inline src/state/GlobalState.tsx:272]: In this case, what happens on the other versions besides appImage?
I also forgot to ask if the information about the updates on the installable version. How are we handling the check if the user is using appImage or has installed heroic in the system?

File: src/state/GlobalState.tsx [hunk 2]
```diff
@@ -266,21 +266,9 @@ export class GlobalState extends PureComponent<Props> {
...
-          'box.appupdate.message',
-          'There is a new version of Heroic Available, do you want to update now?'
-        ),
-        title: t('box.appupdate.title', 'Update Available')
-      })
-
-      if (response === 0) {
-        ipcRenderer.send('openReleases')
-      }
+      ipcRenderer.send('updateHeroic')
```
Comment (flavioislima) [inline src/state/GlobalState.tsx:271]: so you will need some kind of check here, to see if its appImage or not. probably seeing the path, if it starts with /tmp/ will return true if appImage. But I don't know about installed appImages, maybe they will go to /opt, then it won't work.

File: src/state/GlobalState.tsx (hunk 1 above)
Comment (dragonDScript) [reply src/state/GlobalState.tsx:272]: See https://stackoverflow.com/questions/66974771/see-build-target-of-electron-builder-in-running-application

Closing

**Review by flavioislima — APPROVED** [review]

File: src/state/GlobalState.tsx (hunk 2 above)
Comment (dragonDScript) [reply src/state/GlobalState.tsx:271]: Hmmm...maybe check if it's not an app image? like, appimages aren't on /usr, they are on home or the hdd

**Pull request closed without merging**